docs/typescript/how-to-list-open-workflow-executions-using-the-client-in-typescript.md

```diff
@@ -0,0 +1,21 @@
+---
+id: how-to-list-open-workflow-executions-using-the-client-in-typescript
+title: How to list open Workflow Executions using the Client in TypeScript
+sidebar_label: List open Workflow Executions using the Client
+description: List open Workflow Executions using the Client
+tags:
+  - developer-guide
+  - typescript
+  - client
+---
+
+You can provide key-value pairs as Search Attributes in [StartWorkflowOptions](https://typescript.temporal.io/api/interfaces/client.WorkflowOptions#searchattributes).
+In TypeScript, SearchAttributes are represented as `Record<string, string | number | boolean>`.
+
+- The value provided in the map must match what is registered in the dynamic config.
+- The type of value should be a primitive (e.g. string, number, boolean), for dates use `Date.toISOString()`.
+
+This can be useful for tagging executions with useful attributes you may want to search up later. For example:
+
+<!--SNIPSTART typescript-search-attributes-at-creation-->
+<!--SNIPEND-->
```

Comment:
Use [`WorkflowService.listOpenWorkflowExecutions`](https://typescript.temporal.io/api/classes/proto.temporal.api.workflowservice.v1.WorkflowService-1#listopenworkflowexecutions):

```ts
import { Connection } from '@temporalio/client'

const connection = await Connection.connect()
await connection.workflowService.listOpenWorkflowExecutions()
```

To search with a [List Filter](/concepts/what-is-a-list-filter/), use [`WorkflowService.scanWorkflowExecutions`](https://typescript.temporal.io/api/classes/proto.temporal.api.workflowservice.v1.WorkflowService-1#scanworkflowexecutions).